content/dstu2/authorization.md

```diff
@@ -9,15 +9,9 @@ title: Authorization | DSTU 2 API
 The Cerner Authorization Server currently supports [OAuth 2.0][1] [SMART on FHIR<sup>®</sup>][4] launch workflows. As a client, it will require interaction between the client (your client application), the user, the authorization server, a SMART Launch server and a resource server that implements the FHIR<sup>®</sup> standard.
 
 ### Registration ###
-In order for your client application to utilize any protected resources, your client application must first register. To do this please [contact us][10] with the following fields, all fields are **required**.
+In order for your client application to utilize any protected resources, your client application must first register using our [code console][10].
```

Comment:
Is there anything they should know about creating a Cerner Care account? I'm going to guess that most won't already have one. I know there's a big blue button, but it may be good to call that out a bit along with any expectations they would need to have (is that immediate?).